docs/guides/signature-verification.md

```diff
@@ -0,0 +1,135 @@
+# Dasharo release signature verification
+
+Each published Dasharo release is signed with a signing key corresponding to
+given platform and versions. The key infrastructure is stored in
+[3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack).
+
+To verify the integrity of the binaries published in release notes on this
+site, please follow the instructions below:
+
+1. Import necessary keys to your keyring:
+
+    ```bash
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/keys/master-key/3mdeb-master-key.asc \
+        -O - | gpg --import -
+
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/dasharo/3mdeb-dasharo-master-key.asc  \
+        -O - | gpg --import -
+
+    wget <release signing key URL> -O - | gpg --import -
+    ```
+
+    - `<release signing key URL>` is provided in the release notes
+
+2. CHeck the signatures on the keys:
+
+    ```bash
+    gpg --check-signatures "Dasharo" "3mdeb"
+    ```
+
+3. Download the binaries, SHA sums and their signature files
+
+    ```bash
+    wget <binary>
+    wget <binary>.sha256
+    wget <binary>.sha256.sig
+    ```
+
+4. Verify the signatures and binary integrity:
+
+    ```bash
+    gpg -v --verify <binary>.sha256.sig <binary>.sha256
+    sha256sum -c <binary>.sha256
+    ```
```

Comment:
If `binary` here is the same as what is above, it means URL, does it works with HTTPS-hosted files?

Reply:
@pietrushnic nope, it must be local file. Otherwise GPG says no such file or directory

Reply:
36ae1fa300d1d09a2be1eda7a146e7dfff436888